homeassistant/components/irish_rail_transport/sensor.py

```diff
@@ -97,8 +97,8 @@ def device_state_attributes(self):
         if self._times:
             next_up = "None"
             if len(self._times) > 1:
-                next_up = self._times[1][ATTR_ORIGIN] + " to "
-                next_up += self._times[1][ATTR_DESTINATION] + " in "
+                next_up = f"{self._times[1][ATTR_ORIGIN]} to "
+                next_up += f"{self._times[1][ATTR_DESTINATION]} in "
                 next_up += self._times[1][ATTR_DUE_IN]
```

Comment:
These three assignments could just be one long assignment within parenthesis. 🤷‍♂️

Reply:
Idk just wanted to avoid one long line, personally I think it looks better 😅

Reply:
We would break it on three lines within  the parenthesis.

Reply:
```py
variable = (
    f"{string1}"
    f"{string2}"
    f"{string3}"
)

Reply:
How could I forget that? 😁 I will fix it hold on.